api/MessageEvent.json

```diff
@@ -18,7 +18,7 @@
             "version_added": "12"
           },
           "firefox": {
-            "version_added": "4"
+            "version_added": "3"
```

Comment:
Some features remain as "4" which could be correct but seems suspect. Can you double check them?

Reply:
Looks like they're all accurate.  Ran the collector in Firefox 3.6 on BrowserStack and double-checked everything still set to Firefox 4!

Reply:
The constructor is the only thing still set to "4" and I've confirmed that it *doesn't* work in Firefox 4, it was added later. Can you find the version with http://mdn-bcd-collector.appspot.com/tests/api/MessageEvent/MessageEvent?

Reply:
Ah, didn't test the other way -- fixed!